docs/src/pages/[platform]/connected-components/storage/storageimage/react.mdx

```diff
@@ -29,7 +29,7 @@ import '@aws-amplify/ui-react/styles.css';
 ```
 </ExampleCode>
 
-At a minimum you must include the `alt` and `path` props. `path` is a full S3 object key and refers to the [Amplify Storage path](https://docs.amplify.aws/react/build-a-backend/storage/download-files/). It is either a `string` or a callback function that accepts the current user's Cognito `identityId` and returns a `string`.
```

Comment:
This url doesn't exist on the current live docs so replacing it with the gen1 link